source/appModuleHandler.py

```diff
@@ -51,6 +51,62 @@
 post_appSwitch = extensionPoints.Action()
 
 
+_executableNamesToAppMods: Dict[str, str] = {
+	# Azure Data Studio (both stable and Insiders versions) should use module for Visual Studio Code
+	"azuredatastudio": "code",
+	"azuredatastudio-insiders": "code",
+	# Windows 11 calculator should use module for the Windows 10 one.
+	"calculatorapp": "calculator",
+	# The Insider version of Visual Studo Code should use the module for the stable version.
+	"code - insiders": "code",
+	# commsapps is an aliast for the Windows 10 mail and calendar.
+	"commsapps": "hxoutlook",

```

Comment:
Actually, you can either:

* Remove commsaps as it is no longer deployed to the public
* Assign commsapps to hxmail as commsapps derives from the empty hxoutlook, which in turn derives from hxmail

Reply:
Thanks - I've done the latter.